rumqttc/src/eventloop.rs

```diff
@@ -327,3 +327,341 @@ pub(crate) async fn next_pending(
     time::sleep(delay).await;
     pending.next()
 }
+
+#[allow(dead_code)]
```

Comment:
needed to add this because we only use this in unit tests in `eventloop` module. We might also use this in other modules so added it here instead of in unit tests module directly.